src/libostree/ostree-repo-commit.c

```diff
@@ -2593,16 +2593,6 @@ write_directory_to_mtree_internal (OstreeRepo                  *self,
 
       ostree_mutable_tree_set_metadata_checksum (mtree, ostree_repo_file_tree_get_metadata_checksum (repo_dir));
```

Comment:
Yeah...there isn't.  In git the metadata is inline in the tree object but for ostree I split it out since it contains xattrs and is potentially a bit larger.  In practice I'm not sure this was a win though.